app/src/main/java/com/github/h3lp3rs/h3lp/forum/FireForum.kt

```diff
@@ -4,55 +4,62 @@ import android.os.Build
 import androidx.annotation.RequiresApi
 import com.github.h3lp3rs.h3lp.database.Databases.Companion.databaseOf
 import com.github.h3lp3rs.h3lp.database.Databases.FORUM
-import com.github.h3lp3rs.h3lp.forum.*
-import com.github.h3lp3rs.h3lp.forum.ForumCategory.*
 import com.github.h3lp3rs.h3lp.forum.ForumCategory.Companion.DEFAULT_CATEGORY
 import com.github.h3lp3rs.h3lp.forum.ForumCategory.Companion.forumOf
+import com.github.h3lp3rs.h3lp.forum.ForumCategory.valueOf
+import com.github.h3lp3rs.h3lp.forum.ForumCategory.values
 import com.github.h3lp3rs.h3lp.forum.data.ForumPostData
 import java.time.ZonedDateTime
 import java.util.concurrent.CompletableFuture
 
-class FireForum(override val path: Path) : Forum {
+class FireForum(override val path : Path) : Forum {
     private val rootForum = databaseOf(FORUM)
 
     @RequiresApi(Build.VERSION_CODES.O)
-    override fun newPost(author: String, content: String): CompletableFuture<ForumPost> {
+    override fun newPost(author : String, content : String) : CompletableFuture<ForumPost> {
         var key = ""
         // Incrementing by 2 so that a post's replies key is always 1 more than a post's key (this
         // is also an optimization to avoid us requiring 2 calls to incrementAndGet)
         return rootForum.incrementAndGet(UNIQUE_POST_ID, 2).thenApply { postKey ->
             key = postKey.toString()
             val repliesKey = (postKey + 1).toString()
-
+            val currentTime = ZonedDateTime.now()
             val forumPostData = ForumPostData(
                 author,
                 content,
-                ZonedDateTime.now(),
+                getFormattedPostTime(currentTime),
```

Comment:
You could have used the java `Date` class instead of implementing your own parser.
Indeed, we know that Date is easily serializable since it was used before. This is actually more robust as you would get the exact time (Zone, Date, Seconds, Millies, etc).

Reply:
<img width="303" alt="Screenshot 2022-05-18 at 3 11 37 PM" src="https://user-images.githubusercontent.com/79541288/169046672-a99e163a-5f19-4615-8bb5-e714b31d07fc.png">